x-pack/plugins/fleet/server/services/preconfiguration.ts

```diff
@@ -365,6 +367,22 @@ export async function ensurePreconfiguredPackagesAndPolicies(
     allPackagePolicyIds.items
   );
 
+  // Ensure `keep_policies_up_to_date` is set to true for any package that appears in `DEFAULT_PACKAGES`
+  const defaultPackages = await getInstallations(soClient, {
+    searchFields: ['name'],
+    search: DEFAULT_PACKAGES.map(({ name }) => name).join('|'),
+  });
+  const defaultPackagesToUpdate = defaultPackages.saved_objects.filter(
+    ({ attributes }) => !attributes.keep_policies_up_to_date
```

Comment:
Do you think it will be easier to force that when we install|update a default package? Reading this I think this could be easier to enforce that rules by moving this to the package service

Reply:
Yeah was just looking at that. I think we can change the default value at install time instead. I will make this change.

Reply:
Moved to `packages/install.ts` in https://github.com/elastic/kibana/pull/116993/commits/204bf12b7038b8483ffc96998638be4bddc67c7c - tested and works well